pyproject.toml

```diff
@@ -63,9 +63,11 @@ simplejson = ">=3.17.0"
 # Optional dependencies for markup extra
 anymarkup-core = { version = ">=0.8.0", optional = true }
 xmltodict = { version = ">=0.12.0", optional = true }
+pytest-container = {version = "*", optional = true}
```

Comment:
maybe add another comment indicating that this is not a dependency for "markup extra"?

Reply:
yes makes sense and fixed